### ISSUE TITLE
new SomeModel().delete() throw SQLiteDoneException

### ISSUE THREAD

**Issue by Honghe** [Raizlabs/DBFlow#1075]
DBFlow Version: 4.0.0-beta2
Issue Kind (Bug):

Description: 
I just `new SomeModel() and delete()`, but DBflow should count the `delete` query result count, and result in SQLiteDoneException.
```
 E/FlowLog: android.database.sqlite.SQLiteDoneException
       at android.database.sqlite.SQLiteConnection.nativeExecuteForLong(Native Method)
       at android.database.sqlite.SQLiteConnection.executeForLong(SQLiteConnection.java:598)
       at android.database.sqlite.SQLiteSession.executeForLong(SQLiteSession.java:652)
       at android.database.sqlite.SQLiteStatement.simpleQueryForLong(SQLiteStatement.java:107)
       at com.raizlabs.android.dbflow.structure.database.AndroidDatabaseStatement.simpleQueryForLong(AndroidDatabaseStatement.java:67)
       at com.raizlabs.android.dbflow.sql.SqlUtils.longForQuery(SqlUtils.java:204)
       at com.raizlabs.android.dbflow.sql.language.BaseQueriable.count(BaseQueriable.java:44)
       at com.raizlabs.android.dbflow.sql.saveable.ModelSaver.delete(ModelSaver.java:122)
       at com.raizlabs.android.dbflow.sql.saveable.ModelSaver.delete(ModelSaver.java:113)
       at com.raizlabs.android.dbflow.structure.ModelAdapter.delete(ModelAdapter.java:168)
       at com.raizlabs.android.dbflow.structure.BaseModel.delete(BaseModel.java:73)
```

And I currently delete row as follow, is it ok?
```
SQLite.delete(SomeModel.class)
        .where(SomeModel_Table.id.eq(someModel.getId()))
        .execute();
```

**Comment by nikacotAndroid** [comment]
I get the same error. Is there any update for this one?

**Comment by trevjonez** [comment]
It is just logging that can be ignored. It is important to keep in mind that you are targeting a beta build.

`FlowLog.setMinimumLoggingLevel(FlowLog.Level.WTF);`

**Comment by agrosner** [comment]
try using `executeUpdateDelete()`

**Comment by agrosner** [comment]
should be fixed in 4.0.0-beta3
